open-grant-proposals/open-proposal-musicstudio.md

```diff
@@ -0,0 +1,90 @@
+
+# Open Grant Proposal: `Music Studio`
+
+**Name of Project: Music Studio**
+
+**Proposal Category:** `app-dev`
+
+**Proposer:** `@immartian`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+Music Studio is based on the established Musicoin Project to enable global indie musicians to manage and release their musical works and offer distributed content sharing and delivering to music listeners like Airbnb for music.
+
+Today's music industry has failed both musicians and listeners for its single pay-wall model requiring music consumers to subscribe monthly but leaving 90% of Internet users can't share the excellent works excluded by major music labels. Eventually the monopoly of the big industry players siphon over 88% of profit in the industry, leaving only 12% of profit to  musicians, however, only 2% of musicians get really well paid(top commercial ones).
+
+98% of 8 million musicians need a solution to manage their works independently and earn their revenue without middlemen; 2 billion music listeners need a solution to remove the pay-wall to enjoy the best creations all around the world without overpaying to digital music tycoons.  Musicoin Project is working on such a solution to make it possible.
+
+## Value
+
+Previously we used IPFS but didn't fully utilize its potential until Filecoin offers a grander vision that makes it more possible.
+
+Once musicians download our studio app(with Filecoin node built-in), they can not only serve their own content but also being a solid( more trust-able) node of the whole network with real utility. Generally, every musician will offer 50-100 high-definition media files for themselves, whilst its possible to store and sync more content from other peers. Additionally, musicians can early Filecoin if they'd love to contribute their spaces. 
+
+![architecture](music_studio_filecoin.png "")
+
+
+The concern of Filecoin for musicians is on its redundancy because we are now serving content with private IPFS nodes, but if we open it to the whole network of Filecoin, we don't know if there are any potential problems which we never tested before.  
+
+Convincing musicians to adopt Filecoin is a challenge, but it's not as much as other projects since there are already over 5000+ musicians from 27 countries, who are quite familiar with our web-based studio tools.
+
+With this basic assurance, musical content can be a perfect showcase of the Filecoin ecosystem.
+
+## Deliverables
+
+We already have musicoin.org (Webapp), also mobile apps(IOS, Android), we are aiming to create a new application on desktop dedicates to musicians called (MUSIC Studio) based on Filecoin, which is the real distributed part running musical content in a peer2peer way. Music Studio will be the exact project we apply for the grant.
```

Comment:
Is Music Studio already in the works? Is this grant to fund the development of the entire application or just to integrate Filecoin?

Reply:
The past life of Music Studio is https://github.com/Musicoin/desktop, based on which we wish to redesign it into Music Studio, it's not a whole new thing from an architectural perspective since the integration is the key change, but actually the view and vision would be total upgraded.